app/controllers/hiring_staff/vacancies/application_controller.rb

```diff
@@ -49,6 +49,10 @@ def review_path(vacancy)
     school_job_review_path(school_id: school.id, job_id: vacancy.id)
   end
 
+  def review_path_with_errors(vacancy)
+    school_job_review_path(school_id: school.id, job_id: vacancy.id, anchor: 'errors', source: 'publish')
```

Comment:
even though school_id is also set for review path, it's actually something we missed removing when the routes were refactored a while ago, as it is no longer required (we use the one stored in the sesion and this could lead to confussion). Can you remove that please.